### DISCUSSION TITLE
aws-lambda forward_request_body parses body in seperate fields.

### DISCUSSION THREAD

**Discussion by xblaatx** [Kong/kong#10435] [Help] [answered]
Hi all.

When forwarding the body to a aws lambda. the body I send gets parsed into seperate fields. Can this be disabled? so i just receive the original body.

example: 

This body is sent to kong.
[
    {
    "id": 1,
    "country": "BE",
    "startDate":"2022-01-01"
    },
    {
    "id": 2,
    "country": "NL",
    "startDate":"2022-01-02"
    },
    {
    "id": 3,
    "country": "LU",
    "startDate":"2022-01-03"
    }
]

and this is send from kong to aws lambda

{
    '1': { id: 1, country: 'BE', startDate: '2022-01-01' },
    '2': { id: 2, country: 'NL', startDate: '2022-01-02' },
    '3': { id: 3, country: 'LU', startDate: '2022-01-03' },
}

**Comment by ghost** [comment]
What is your `aws-lambda` plugin configuration? and how you are sending the request.

**Reply by ghost** [reply]
what `content-type` in your request header which send to Kong?

**Comment by xblaatx** [comment, marked as answer]
I found out that the request transformer plugin was transforming the object. So it has nothing to do with the aws-lambda plugin. 
Problem solved :-)
